RELEASE_NOTES.rst

```diff
@@ -79,6 +92,12 @@ Bugfixes:
 
 For bear writers:
 
+-  A new built-in type is available from ``Setting`` for using inside ``run()``
+   signature: ``url``.
+-  ``Lint`` based bears have a new argument which can be set to test whether a
+   command runs without errors. This can be used to check for example the
```

Comment:
this still doesnt sound right imo, its either "Example: This can be used to check for the existence of a Java module."
OR: "This can be used to check, for example, the existence of a Java module."